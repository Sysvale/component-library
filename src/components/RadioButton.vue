<template>
	<span id="radioButton">
		
	</span>
</template>

<script>
export default {
	data() {
		return {
		}
	},
};
</script>
<style>
</style>
