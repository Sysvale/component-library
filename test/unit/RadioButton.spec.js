import { mount, createLocalVue } from '@vue/test-utils';
import RadioButton from '../../src/components/RadioButton.vue';

import BootstrapVue from 'bootstrap-vue';
const localVue = createLocalVue();
localVue.use(BootstrapVue);

const options = [
	{
		text: 'Component1',
		id: 'id1',
		disabled: true,
	},
	{
		text: 'Component2',
		id: 'id2',
	},
	{
		text: 'Component3',
		id: 'id3',
	},
];

test('Component is mounted properly', () => {
	const wrapper = mount(RadioButton, {
		localVue,
		propsData: {
			value: '',
			options: options,
		},
	});

	expect(wrapper).toMatchSnapshot();
});
