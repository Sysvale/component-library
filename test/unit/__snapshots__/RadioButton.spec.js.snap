// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component is mounted properly 1`] = `<span id="radioButton"><label for="id" class="radio-button-container"><input type="radio" id="id" value="id"> <label for="id" class="m-0">test</label></label></span>`;
